e2e/detail.test.ts

```diff
@@ -0,0 +1,24 @@
+import { by, device, element, expect } from 'detox';
+
+describe('Detail', () => {
+    beforeAll(async () => {
+        await device.launchApp();
+    });
+
+    beforeEach(async () => {
+        await device.reloadReactNative();
+    });
+
+    it('should display detail screen', async () => {
+        await element(by.id('Menu')).tap();
+        await element(by.id('Detail')).tap();
+        await expect(element(by.id('Detail'))).toBeVisible();
+    });
+
+    it('should change index when pressing Next button', async () => {
+        await element(by.id('Menu')).tap();
+        await element(by.id('Detail')).tap();
+        await element(by.id('Next')).tap();
+        await expect(element(by.id('2'))).toBeVisible();
```

Comment:
But what I really want to see is the following:
- Check that the text for the letter is rendered.
- Check that the text for the word is rendered.
- Check that the image for the given letter is rendered.
With a TODO to
- Check that when you click the tile area it plays the word audio
- Check that when you click the word text area it plays the word audio
- Check that when you click the letter text area it plays the letter audio
Note that you will wait for a future store to implement this part.

Reply:
And to make this workout, I think it's a good time to build some dummy data. Can you pull down the production data in JSON and then make a second `english-alphabet.json` that has the same structure, but uses the English alphabet with each English letter and a sample word for each? Invent placeholder URLs for audio and images for now, but we can tackle getting some real data once you get to linking audio and images. You can use this dummy data in your test expectations. So if 'b' and 'banana' are the second letter and word, you can scroll to 'b' and see that you find 'banana'.